### PR TITLE
Don't install x86 binaries on arm architectures

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -223,48 +223,51 @@ parts:
...
     source: https://zoom.us/client/latest/zoom_amd64.deb
     override-build: |
-      # remove danging symlink
-      rm -f $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      echo "QT_QPA_PLATFORM=xcb \$SNAP/opt/zoom/ZoomLauncher" > $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      chmod +x $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      # Unfrig LD_LIBRARY_PATH
-      sed -i s/D_LIBRAR/XXXXXXXX/g opt/zoom/ZoomLauncher
-      snapcraftctl build
+      if [[ `arch` = "x86_64" ]]; then
```
Comment (Saviq) [inline snap/snapcraft.yaml:226]: Better to use `${SNAPCRAFT_ARCH_TRIPLET}` here?

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -223,48 +223,51 @@ parts:
...
+        # remove danging symlink
```
Comment (Saviq) [inline snap/snapcraft.yaml]:
```suggestion
        # remove dangling symlink
```

Since you're touching this…

**Review by Saviq — CHANGES_REQUESTED** [review]
What will happen with the `zoom` app outside of amd64? It just won't be there?

File: snap/snapcraft.yaml (hunk 1 above)
Comment (AlanGriffiths) [reply snap/snapcraft.yaml:226]: :shrug: I copied one of popey's examples. Does it matter?

**Comment by AlanGriffiths** [comment]
> What will happen with the `zoom` app outside of amd64? It just won't be there?

Yes, better than having the amd64 binary. (Which doesn't work)

**Comment by Saviq** [comment]
> Yes, better than having the amd64 binary. (Which doesn't work)

Sorry, I meant in egmde, it will just not show up in the list?

Feel free to merge if that's the case.

bors delegate+

**Comment by AlanGriffiths** [comment]
> Sorry, I meant in egmde, it will just not show up in the list?

Yes

> bors delegate+

No bors here

**Pull request merged**